app/models/host_status/build_status.rb

```diff
@@ -64,6 +64,21 @@ def token_expired?
     def build_errors?
       host && host.build_errors.present?
     end
+
+    def remediation_help_text
+      case to_status
```

Comment:
This should use `status`. `to_status` can call expensive (external services). `status` just used the cached version in the database.